plugins/modules/ipaservice.py

```diff
@@ -839,6 +839,10 @@ def main():
                         changed = True
                 else:
                     changed = True
+            except ipalib.errors.EmptyModlist:
```

Comment:
Please use `ipalib_errors` from `ansible_freeipa_module` instead of `ipalib.errors`.

This needs to be fixed in other modules also.

Reply:
Fixed.